hspy_bundle/NSIS_installer_script.nsi

```diff
@@ -307,7 +307,7 @@ SectionIn RO
 	File /r "${WINPYTHON_PATH}\*"
 	${If} $InstMode = 2
 	; Create right-click context menu entries for Hyperspy Here
-		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython Command Prompt.exe" install_hyperspy_here & exit"'
+		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython Command Prompt.exe" jupyter-context_menu-add & exit"'

```

Comment:
Are you suggesting to change it to jupyter_context-menu_add? That would
make sense. However, I've gone for the other way around to make it
easier to discover the command, because all jupyter commands are
jupyter-whatever...


On Thu, 2016-02-18 at 02:56 -0800, Vidar Tonaas Fauske wrote:
> In hspy_bundle/NSIS_installer_script.nsi:
> > @@ -307,7 +307,7 @@ SectionIn RO
> >  	File /r "${WINPYTHON_PATH}\*"
> >  	${If} $InstMode = 2
> >  	; Create right-click context menu entries for Hyperspy
> Here
> > -		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython
> Command Prompt.exe" install_hyperspy_here & exit"'
> > +		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython
> Command Prompt.exe" jupyter-context_menu-add & exit"'
> Just as a nitpick, I always translate underscores as spaces, so I
> read this as "jupyter-context menu-add".
> —
> Reply to this email directly or view it on GitHub.